descqa/ColorDistribution.py

```diff
@@ -60,7 +60,8 @@ def __init__(self, **kwargs): # pylint: disable=W0231
         self.color_transformation_q = kwargs.get('color_transformation_q', True)
         self.Mag_r_limit = kwargs.get('Mag_r_limit', None)
         self.rest_frame = kwargs.get('rest_frame', bool(self.Mag_r_limit and not self.obs_r_mag_limit))
-
+        self.use_agn = kwargs.get('use_agn', True)
```

Comment:
Setting default value to `True` can cause confusion, as users usually expect that, when `use_agn` is not set, it means not using AGN. 

A better way would be to rename this argument to `exclude_agn`, which then can be default to `False`.

Reply:
I changed this as you suggested